lms/djangoapps/teams/csv.py

```diff
@@ -306,6 +310,20 @@ def validate_user_assignment_to_team_and_teamset(self, row):
                 return False
         return True
 
+    def validate_compatible_enrollment_modes(self, user, team_name, teamset_id):
```

Comment:
I generally prefer a naming convention that reads better when being invoked (so here, something like `are_enrollment_modes_compatible`). It is following the convention of the file, but I don't like the convention 🙃 

Just my 2 cents.

Reply:
I would keep it for the sake of uniformity, but keep it in mind for future. Can be swayed either way.